Source/WebCore/html/shadow/attachmentElementShadow.css

```diff
@@ -81,6 +81,18 @@ attachment#attachment-preview {
     overflow: hidden;
 }
 
+div#attachment-placeholder {
+    grid-row: 1;
+    grid-column: 1;
+    width: 100%;
+    aspect-ratio: 1;
+    /* SF Symbol arrow.down.circle */
```

Comment:
Can you, when importing symbols, specify the size and weight too in your comment?

Reply:
I got them from https://copysfsymbols.apple.com/index.html, no size options there! I'll add a note...

Reply:
We can probably say they are "Medium", since that's the default.